crates/ty_python_semantic/resources/mdtest/import/module_getattr.md

```diff
@@ -0,0 +1,79 @@
+# Module-level `__getattr__`
```

Comment:
Great thorough tests!

It would be good to also have a test showing the precedence with submodules; that is, that if `mod/__init__.py` has a `__getattr__`, and there also exists a `mod/sub.py`, that if we have `import mod; reveal_type(mod.sub)` we see the return type of `__getattr__`, but if we have `import mod.sub; reveal_type(mod.sub)` we see the submodule.

(I already checked this works correctly in this PR; it would just be ideal to test it.